Kernel/SanCov.cpp

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright (c) 2021, Patrick Meyer <git@the-space.agency>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#include <Kernel/Devices/KCOVDevice.h>
+#include <Kernel/Devices/KCOVInstance.h>
+#include <Kernel/FileSystem/FileDescription.h>
+#include <LibC/sys/ioctl_numbers.h>
```

Comment:
Seems like most of these headers aren't used in this file? 

Probably only need... one for Processor::current().in_irq(), Thread.h, and KCovDevice/Instance.h?

Reply:
Ah sorry for this. I changed this to:
```
#include <Kernel/Devices/KCOVDevice.h>
#include <Kernel/Process.h>
#include <Kernel/Thread.h>
```

As I use those Namespaces directly in this file.

Reply:
Cleaned up some more includes while at it.